modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -2492,6 +2492,11 @@ TEST_P(Test_ONNX_layers, OpenAI_CLIP_head)
     testONNXModels("clip-vit-base-head");
 }
 
+TEST_P(Test_ONNX_layers, where_node)
+{
+    testONNXModels("where_layer");
+}
```

Comment:
Please add a test for broadcasting scenario.

Reply:
<img width="328" alt="image" src="https://user-images.githubusercontent.com/16487352/236370792-98c1879c-4b39-4711-9c7c-3fc419fd7593.png">

Hi @dkurt. Thanks for your code review, and I have updated the test case.